packages/orchestrator/lib/routes/v1/taskId/output.ts

```diff
@@ -0,0 +1,54 @@
+import { z } from 'zod';
+import type { Json, Scheduler } from '@nangohq/scheduler';
+import type { ApiError } from '@nangohq/types';
+import type { EndpointRequest, EndpointResponse, RouteHandler, Route } from '@nangohq/utils';
+import { validateRequest } from '@nangohq/utils';
+
+interface Output {
```

Comment:
I feel like it's more handy to use Endpoint so you get autocompletion and error there. Otherwise it feels like a free form object and fails everywhere else but here
```suggestion
type Output = Endpoint<{
```